tekton-chains/base/clusterrole.yaml

```diff
@@ -0,0 +1,47 @@
+kind: ClusterRole
```

Comment:
Any reson why we chose to put this here? Roles are RBAC which are always scoped to the cluster or cluster resource, even if they are application specific. Usually what we do is create a bundle `cluster-scope/bundles` that references all the other cluster scoped resources like `clusterrolebindings`, `clusterroles`, `roles`, `rolebindings`, `operatorGroups`, `subscriptions`, etc. We then reference that bundle from either the application overlay or the cluster-scope overlay (which becomes cluster-resources in argocd).